src/Http/Shared/StreamCopyOperationInternal.cs

```diff
@@ -42,13 +43,13 @@ public static async Task CopyToAsync(Stream source, Stream destination, long? co
             {
                 Debug.Assert(source != null);
                 Debug.Assert(destination != null);
-                Debug.Assert(!bytesRemaining.HasValue || bytesRemaining.GetValueOrDefault() >= 0);
+                Debug.Assert(!bytesRemaining.HasValue || bytesRemaining.Value >= 0);
                 Debug.Assert(buffer != null);
 
                 while (true)
                 {
                     // The natural end of the range.
-                    if (bytesRemaining.HasValue && bytesRemaining.GetValueOrDefault() <= 0)
+                    if (bytesRemaining.HasValue && bytesRemaining.Value <= 0)
```

Comment:
Why did you change these to `Value`? Value checks whether the value is null and throws an error if it is. `GetValueOrDefault` returns whatever the value is, making it faster.